src/types/Privacy.ts

```diff
@@ -48,21 +50,28 @@ export type PrivacyType = {
      *
      * @param enabled true to enable the MAX Terms Flow.
      */
-    setConsentFlowEnabled(enabled: boolean): Promise<void>;
+    setConsentFlowEnabled(enabled: boolean): void;
```

Comment:
I think we should just deprecate this and add a new API. This was the API for old `Terms Flow`. Right now, if a pub is using TermsFlow, they'd automatically be moved to the new flow. 

We can probably call the new API `setTermsAndPrivacyPolicyFlowEnabled(boolean)`.

Reply:
Has been added `setTermsAndPrivacyPolicyFlowEnabled(boolean)`.  Please review.